core/domain/exp_domain.py

```diff
@@ -3368,8 +3386,9 @@ def _choices_should_be_unique_and_non_empty(
                 'translations_mapping'][content_id]
             for translation in choice_translations.values():
                 translation['needs_update'] = True
-
-            choice_voiceovers = state_dict['recorded_voiceovers'][
+            # Here we use MyPy ignore because the latest schema of state
+            # dict doesn't contains recorded_voiceovers property.
+            choice_voiceovers = state_dict['recorded_voiceovers'][ # type: ignore[misc]
```

Comment:
I may be missing something but will this not create any issue for the new version of the exploration as they will not contain `recorded_voiceovers`?

Reply:
Got it, make sense.